subprojects/execution/src/test/groovy/org/gradle/internal/execution/steps/BuildCacheStepTest.groovy

```diff
@@ -127,7 +130,7 @@ class BuildCacheStepTest extends StepSpec<IncrementalChangesContext> implements
 
         then:
         def ex = thrown Exception
-        ex.message == "Failed to load cache entry for job ':test'"
+        ex.message == "Failed to load cache entry $cacheKeyHashCode from $cacheName for job ':test': unpack failure"
```

Comment:
Instead of passing the `cacheName` as a parameter, it would be better to pass the whole expected message. That way we can actually verify what the user will see directly, which is less error-prone.

Reply:
I changed the `$cacheKeyHashCode` to literal. But `$cacheName` is part of `where:` block, so I can hardcode it.

Reply:
Fixed after removing `BuildCacheLoadException`.